src/driver/better-sqlite3/BetterSqlite3Driver.ts

```diff
@@ -48,6 +51,15 @@ export class BetterSqlite3Driver extends AbstractSqliteDriver {
     // Public Methods
     // -------------------------------------------------------------------------
 
+    async connect() {
+        await super.connect()
+
+        this.version = this.databaseConnection
```

Comment:
We usually query the version using `QueryRunner.getVersion()`.

Maybe it could be done generically for all SQLite-based drivers.